execution.bs

```diff
@@ -4259,14 +4230,85 @@ enum class forward_progress_guarantee {
 
 3. If `this_thread::execute_may_block_caller(s)` for some scheduler `s` returns `false`, no `execution::execute(s, f)` call with some invocable `f` shall block the calling thread.
 
+#### `execution::get_completion_scheduler` <b>[exec.queries.completion_scheduler]</b> #### {#spec-execution.queries.get_completion_scheduler}
+
+1. `execution::get_completion_scheduler` is used to ask a sender object for the <i>completion scheduler</i> for one of its signals.
```

Comment:
Instead of asking a sender, which we no longer do, we should say something like:
> `execution::get_completion_scheduler` obtains <i>completion scheduler</i> for one of  signals from sender attributes.

Reply:
And some kind of changes everywhere